app/views/presentations/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Edit Presentation</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Back', presentations_path %>
```

Comment:
Good point! I used the the previously created people index view as a placeholder to give us easier  access to the edit form and delete link. We may want to add a new story to pivotal to format and style desired views.  I have removed the Back link.